src/Build/Graph/ProjectGraph.cs

```diff
@@ -14,7 +16,10 @@ namespace Microsoft.Build.Graph
     public sealed class ProjectGraph
     {
         private const string ProjectReferenceItemName = "ProjectReference";
-        private const string FullPathString = "FullPath";
+        private const string FullPathMetadataName = "FullPath";
+        private const string ProjectReferenceTargetsItemType = "ProjectReferenceTargets";
+        private const string ProjectReferenceTargetsMetadataName = "Targets";
+        private const string DefaultTargetsMarker = ".default";
```

Comment:
I decided to go with ".default" as its not a valid target name and doesn't "look like" a property or item access.